boards/px4/sitl/default.cmake

```diff
@@ -5,6 +5,7 @@ px4_add_board(
 	MODEL sitl
 	ROMFSROOT px4fmu_common
 	LABEL default
+	EMBEDDED_METADATA parameters
```

Comment:
What if we just did this by default everywhere unless `CONSTRAINED_FLASH`?

Reply:
It's a possibility, I'm not sure yet how the end result should look like. For now I wanted something to selectively enable for testing w/o affecting other targets.
I can imagine that some targets might want to use external metadata (e.g. from a companion) for faster download.